docs/organizations/using-gate-policies.md

```diff
@@ -0,0 +1,92 @@
+# Using gate policies
+
+Gate policies help you ensure that Codacy uses the same quality gates across your organization repositories.
+
+Codacy provides a built-in gate policy, **Codacy gate policy**, with the default quality gates that define when your pull requests and commits meet the minimum quality levels.
+
+This built-in **Codacy gate policy** is primarily the default gate policy, which Codacy applies automatically to newly added repositories. You can then create new gate policies with different quality gates and make them the default for your organization.
```

Comment:
The part "This built-in **Codacy gate policy** is primarily the default gate policy [...]" doesn't really add new information to the previous paragraph. It should be possible to simplify the two paragraphs explaining the built-in gate policy in a more straightforward way.

Here's a simplified version proposed by ChatGPT that you can potentially use as inspiration. :robot: :sweat_smile: 

> Codacy offers a built-in gate policy called Codacy gate policy, which sets minimum quality levels for pull requests and commits. By default, this gate policy is automatically applied to new repositories. Additionally, users have the flexibility to create new gate policies with custom quality gates and set them as the default for their organization.

Reply:
Updated!